app/parseBoard.py

```diff
@@ -106,14 +107,52 @@ def bullyPathing(converted_data, pathBoard):
                     y = a['y']
                     if(x < converted_data['board']['width'] and x > 0):
                         if(pathBoard[y][x+1] != -1):
-                            board[y][x+1] = 3
+                            board[y][x+1] = SMALLER_SNAKE_FUTURE_HEAD
                         if(pathBoard[y][x-1] != -1):
-                            board[y][x-1] = 3
+                            board[y][x-1] = SMALLER_SNAKE_FUTURE_HEAD
                     if(y < converted_data['board']['width'] and y > 0):
                         if(pathBoard[y+1][x] != -1):
-                            board[y+1][x] = 3
+                            board[y+1][x] = SMALLER_SNAKE_FUTURE_HEAD
                         if(pathBoard[y-1][x] != -1):
-                            board[y-1][x] = 3
+                            board[y-1][x] = SMALLER_SNAKE_FUTURE_HEAD
+    return board
+
+
+def cowardPathing(converted_data, pathBoard):
+    """
+    Finds Snakes that are LARGER than us and assigns the area around their head with the LARGER_Snake_Future_Head
+
+    May need logical update for snake bodies but it should be fine
+
+    Args:
+        converted_data (json): python readable json
```

Comment:
(list) not (json)

Reply:
Wait really? We've been calling this Json in some other documentation.... I thought it was a dictionary not a list.

Reply:
*(dict) not (json) my bad 😂

Reply:
Json isn't a valid python type. Python Documentation requires a valid python types

Reply:
Ok. Well that's a change that need to be reflected other places as well.

Reply:
Ill jump on it in a bit